shaky/src/main/java/com/linkedin/android/shaky/SelectFragment.java

```diff
@@ -31,9 +34,27 @@
  */
 public class SelectFragment extends Fragment {
 
+    private static final String KEY_THEME = "theme";
+
+    @Nullable @StyleRes Integer theme;
+
+    static SelectFragment newInstance(@Nullable @StyleRes Integer theme) {
+        SelectFragment fragment = new SelectFragment();
+        Bundle args = new Bundle();
+        if (theme != null) {
+            args.putInt(KEY_THEME, theme);
+        }
+        fragment.setArguments(args);
+        return fragment;
+    }
+
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        @StyleRes int theme = getArguments().getInt(KEY_THEME, FeedbackActivity.MISSING_RESOURCE);
```

Comment:
this change is repeating in multiple file. how about creating a base fragment for this?

Reply:
Created util